apiary.apib

```diff
@@ -1445,11 +1445,11 @@ Get details for a single invoice.
                         + id: `f88ab83c-b8ca-497b-a2c6-1ad631fe1cea` (string)
             + discounts (array)
                 + (object)
-                    + description: `Discount` (string)
                     + type: `percentage` (enum[string])
                         + Members
                             + percentage
                     + value: 15.00 (number)
+                    + discount: `Discount` (string)
```

Comment:
Q: Shouldn't this be description?

Reply:
yes, thanks :)

 :ok: by @luisabritotl on 26/03/2018 15:27:20